addons/l10n_fr/models/res_company.py

```diff
@@ -10,12 +10,21 @@ class ResCompany(models.Model):
     l10n_fr_closing_sequence_id = fields.Many2one('ir.sequence', 'Sequence to use to build sale closings', readonly=True)
     siret = fields.Char(related='partner_id.siret', string='SIRET', size=14, readonly=False)
     ape = fields.Char(string='APE')
+    is_france_country = fields.Boolean(
+        compute="_compute_is_france_country",
+        string="Is Part of DOM-TOM",
+    )
+
+    @api.depends('country_code')
+    def _compute_is_france_country(self):
+        for company in self:
+            company.is_france_country = company.country_code in self._get_france_country_codes()
 
     @api.model
     def _get_france_country_codes(self):
         """Returns every country code that can be used to represent France
         """
-        return ['FR', 'MF', 'MQ', 'NC', 'PF', 'RE', 'GF', 'GP', 'TF'] # These codes correspond to France and DOM-TOM.
+        return ['FR', 'MF', 'MQ', 'NC', 'PF', 'RE', 'GF', 'GP', 'TF', 'BL', 'PM', 'YT', 'WF']  # These codes correspond to France and DOM-TOM.
```

Comment:
When I look at this [page](https://fr.wikipedia.org/wiki/France_d%27outre-mer#Liste), it seems that Clipperton doesn't have any country code. But on this [one](https://en.wikipedia.org/wiki/ISO_3166-2:FR#:~:text=FR%2DCP-,Clipperton,-Clipperton), it seems it's `CP`. Shouldn't we add it? 
cc @Olivier-olma

Reply:
### As the population of Cliperton is estimated to 0 people, and as we don't have the country code in Odoo, I think that we can do without Cliperton